lib/bytesToUuid.js

```diff
@@ -10,14 +10,14 @@ for (var i = 0; i < 256; ++i) {
 function bytesToUuid(buf, offset) {
   var i = offset || 0;
   var bth = byteToHex;
-  return bth[buf[i++]] + bth[buf[i++]] +
-          bth[buf[i++]] + bth[buf[i++]] + '-' +
-          bth[buf[i++]] + bth[buf[i++]] + '-' +
-          bth[buf[i++]] + bth[buf[i++]] + '-' +
-          bth[buf[i++]] + bth[buf[i++]] + '-' +
-          bth[buf[i++]] + bth[buf[i++]] +
-          bth[buf[i++]] + bth[buf[i++]] +
-          bth[buf[i++]] + bth[buf[i++]];
+  return ([bth[buf[i++]], bth[buf[i++]], 
```

Comment:
Please add a comment here referencing https://bugs.chromium.org/p/v8/issues/detail?id=3175#c4 as the reason for using Array#join() here, then I'll merge this.